skiko/src/awtMain/kotlin/org/jetbrains/skiko/SkiaLayer.awt.kt

```diff
@@ -212,6 +212,8 @@ actual open class SkiaLayer internal constructor(
     /**
      * Returns the pointer to an OS specific window handle (native resource)
      * which the current [SkiaLayer] is attached.
+     * On macOS accessing this property returns an owning raw pointer to the NSWindow.
+     * Failure to pass this pointer to consuming APIs will result in a memory leak.
```

Comment:
`windowHandle` is a public API in Compose.

I have concerns with the current approach:
- calling it requires users to think about how to transfer ownership
- not sure, but it probably prohibits calling it multiple times
- when we pass it to `waitVSync`, can it still be accessible by user?
- on other systems, calling this doesn't transfer ownership - the window is only valid until it isn't closed

Can we fix the internal issue without changing ownership in this call? For example, we can take ownership on a redrawer creation and release it on `dispose`.